ddtrace/llmobs/_evaluators/runner.py

```diff
@@ -66,12 +64,14 @@ def start(self, *args, **kwargs):
             return
         super(EvaluatorRunner, self).start()
         logger.debug("started %r to %r", self.__class__.__name__)
-        atexit.register(self.on_shutdown)
 
-    def stop(self, *args, **kwargs):
-        if self.status == service.ServiceStatus.STOPPED:
-            return
-        super(EvaluatorRunner, self).stop()
+    def _stop_service(self) -> None:
+        """
+        Ensures all spans are evaluated & evaluation metrics are submitted when evaluator runner
+        is stopped by the LLM Obs instance
+        """
+        self.periodic(_wait_syncronously=False)
```

Comment:
i might have this wrong, but do we want `_wait_syncronously=True` here? re: 
```
This param is only set to True for when the evaluator runner is stopped by the LLM Obs instance on process exit and we want to block until all spans are evaluated and metrics are submitted.
```
below. i thought this `_stop_service` was called from that disable at exit?

Reply:
Thanks for catching! I must of flipped it when I was testing (fix attempt 1)